ts/src/header-validator/validate-json.ts

```diff
@@ -1194,6 +1194,10 @@ function aggregatableTriggerData(
   )
 }
 
+export type AggregatableValue = FilterPair & {
```

Comment:
Let's rename this type to whatever we end up going with in the spec per my other comment.